playbook/app/pb_kits/playbook/pb_button/_button.scss

```diff
@@ -1,7 +1,24 @@
 @import "./button_mixins";
 
+$pb_button_sizes: (
+  "sm":   0.75rem,
+  "md":   0.875rem,
+  "lg":   1.125rem,
+);
+
 [class^=pb_button_kit]{
   @include pb_button;
+  // Size =================
+  @each $name, $size in $pb_button_sizes {
+    &[class*=_#{$name}] {
+      font-size: $size;
+      padding: calc(#{$size} / 2) calc(#{$size} * 2.42) !important;
```

Comment:
This is very cool, but I do have just one question on the origin of `2.42` within the calculated x padding here. Is there any reference to point to? Mostly I'm just curious. On the `!important` part, what would make this necessary? Which other selector(s) are making this necessary?

Reply:
Jon suggested to convert size values from px to rems so we can calculate the padding.  `!imprtant`is used to override the previous styles on this specific property.